Make GridTrackSize import in entities.ts relative

Make the GridTrackSize import in [entities.ts](https://github.com/DataBiosphere/findable-ui/blob/2b986bb0a647019f19653cb0ba758b4778f74267/src/common/entities.ts) relative.

Fixes: `Cannot find module 'config/entities' or its corresponding type declarations.`

--- a/src/common/entities.ts
+++ b/src/common/entities.ts
@@ -1,5 +1,5 @@
 import { ColumnDef } from "@tanstack/react-table";
-import { GridTrackSize } from "config/entities";
+import { GridTrackSize } from "../config/entities";
 
 /**
  * Model of a value of a metadata class.

